queueing/consumer.py

```diff
@@ -145,16 +93,16 @@ def consume_from_kafka(topic_name):
 
         collection_name = "{}-{}-{}".format(spider, job, topic_name)
         identifier = "{}/{}".format(project, collection_name)
-        unique = message.value.get("unique", "False") == "True"
+        unique = message.value.get("unique", "") == "True"
```

Comment:
Why is this changing? Doesn't the message contain `"True"` or `"False"`?

Reply:
The messages of the _requests_ and _logs_ don’t have the field `unique`. Furthermore, the functionality is the same.